deployment_config/dev_vars.yml

```diff
@@ -1 +1,8 @@
+AUTH_BASE: https://uat.hsesinfo.org
+AUTH_CLIENT_ID: ((AUTH_CLIENT_ID))
+AUTH_CLIENT_SECRET: ((AUTH_CLIENT_SECRET))
+env: dev
 instances: 1
+NODE_ENV: production
+REDIRECT_URI_HOST: https://tta-smarthub-dev.app.cloud.gov
+SESSION_SECRET: ((SESSION_SECRET))
```

Comment:
Future cleanup task: I'm 99% certain that `AUTH_CLIENT_ID` `AUTH_CLIENT_SECRET` and `SESSION_SECRET` don't need to be in these files, so I'd like to remove them, but it's not important enough to hold this merge up.

Reply:
I really hope they're not needed too. I'm very happy you're okay with just moving on for now. :)